src/backend/joanie/core/api/admin.py

```diff
@@ -308,3 +308,20 @@ class ContractDefinitionViewSet(viewsets.ModelViewSet):
     permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
     serializer_class = serializers.AdminContractDefinitionSerializer
     queryset = models.ContractDefinition.objects.all()
+
+
+class OrderGroupAccessViewSet(
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    """
+    OrderGroup ViewSet
+    """
+
+    authentication_classes = [SessionAuthenticationWithAuthenticateHeader]
+    permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
+    serializer_class = serializers.AdminOrderGroupSerializer
+    queryset = models.OrderGroup.objects.all().order_by("id")
```

Comment:
Why do you need to use `order_by("id")` ?

Reply:
To avoid some warnings but it is not functionally mandatory

Reply:
Oh ok. There is a class attribute for that:

```python
ordering = ["-created_on"]
```

Reply:
Is it worth to make a migration for a warning? (putting ordering = ["-created_on"] in thge viewset does not get rid of the warning)